packages/app/src/cli/services/app/config/link.ts

```diff
@@ -126,7 +134,7 @@ async function loadConfigurationFileName(
 
 export function mergeAppConfiguration(
   appConfiguration: AppConfiguration,
-  remoteApp: OrganizationApp,
+  remoteApp: OrganizationApp & Pick<CurrentAppConfiguration, 'access'>,
```

Comment:
this won't scale either. Can we have an Schema with all the app-config fields and use it here? (that schema will only have `access` for now)

Reply:
Is this close to what you have in mind?

```ts
type ApiAccessMode =
  | true
  | {
      mode: 'offline' | 'online'
    }

type OrganizationConfigurationApp = OrganizationApp & {
  access: {
    api_access: ApiAccessMode
  }
}
```

Reply:
✅ `OrganizationConfigurationApp` schema has been added in 5ea660f904de69f94fe20b17894f775b826474d2. 

Here's the updated schema with the relevant Zod `AppSchema` code:

```ts
type ApiAccessMode =
  | true
  | {
      mode: 'offline' | 'online'
    }

export type OrganizationConfigurationApp = OrganizationApp & {
  access?: {
    api_access?: ApiAccessMode
  }
}

export const AppSchema = zod
  .object({
    access: zod
      .object({
        api_access: zod.union([zod.literal(true), zod.object({mode: zod.enum(['online', 'offline'])})]).optional(),
      })
      .optional(),
  })
  .strict()
```